src/main/resources/assets/idwtialsimmoedm/lang/ru_ru.json

```
@@ -0,0 +1,47 @@
+{
+  "text.idwtialsimmoedm.config.title": "Настройки idwtialsimmoedm",
+  "text.idwtialsimmoedm.config.field.descriptionPrefix": "Префикс перед описанием (1 линия)",
+  "text.idwtialsimmoedm.config.field.descriptionIndent": "Отступы слева от описания (2+ линия)",
```

То же самое. "Описание" можно опустить, так как текст логически связан с предыдущей настройкой.
Само слово "отступ" подразумевает в том числе боковое смещение, поэтому "слева от" также не обязательно.
```suggestion
  "text.idwtialsimmoedm.config.field.descriptionIndent": "Отступы последующих строк",
```